From: Antoine Musso <hashar@free.fr>
Date: Mon, 4 Nov 2013 12:17:58 +0100
Subject: wmf: soften requirements

WebOb and paramiko have a minimal version which is enforced by OpenStack
but is probably not really needed.

ordereddict and argparse are included in python 2.7 and we do not care
about python 2.6.

Change-Id: Ie2bffccd0b4aa8b505a5c7de8174a42d4395d9d7
---
 requirements.txt | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index c682999..e82ad04 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -3,10 +3,9 @@ pbr>=0.5.21,<1.0
 argparse
 PyYAML>=3.1.0
 Paste
-WebOb>=1.2.3,<1.3
-paramiko>=1.8.0
+WebOb
+paramiko
 GitPython>=0.3.3
-ordereddict
 python-daemon>=2.0.4
 extras
 statsd>=1.0.0,<3.0
